src/coffee/card.coffee

```diff
@@ -196,6 +201,19 @@ class Card
     QJ.toggleClass el, @options.classes.valid, test
     QJ.toggleClass el, @options.classes.invalid, !test
 
+  maskCardNumber: (val, el, out) =>
+    mask = @options.masks.cardNumber
+    cardType = payment.fns.cardType(val)
+    numbers = val.split(' ')
+
+    if (cardType == 'amex' && (numbers.length == 3)) || numbers.length >= 4
```

Comment:
Can you explain why we have to add this exception for amex? I'm a little bit wary of adding card type specific code here and wonder whether we can come up with a general solution.